src/main/java/pl/pop/interview/master/answer/AnswerController.java

```diff
@@ -0,0 +1,27 @@
+package pl.pop.interview.master.answer;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.pop.interview.master.question.QuestionDTO;
+
+@RestController
+@RequestMapping("api/v1/answers")
```

Comment:
wiem, że się czepiam już teraz, ale jakby to też się dało uporządkować, żeby estetycznie to wyglądało, to by było super <3

Reply:
Bardzo mi się podoba taka dbałość. Done.